test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/ServiceBusEndToEndTests.cs

```diff
@@ -248,16 +309,16 @@ private async Task ServiceBusEndToEndInternal<T>(IHost host = null)
                     $"{jobContainerType.FullName}.SBTopicListener2",
                     $"{jobContainerType.FullName}.ServiceBusBinderTest",
                     "Job host started",
-                    $"Executing '{jobContainerType.Name}.SBQueue2SBQueue' (Reason='New ServiceBus message detected on '{FirstQueueName}'.', Id=",
```

Comment:
This is an unfortunate loss. EventHub doesn't have a Reason either? Assume this is because the "strategy" pattern doesn't allow you to specify your own ParameterDescriptor, which means you can't implement GetTriggerReason

Reply:
Yes, EventHub does not have a Reason. Cosmos trigger creates `CosmosDBTriggerBinding`, ServiceBus/EventHub create `ITriggerBinding` using `BindingFactory.GetTriggerBinding(new ServiceBusTriggerBindingStrategy(), parameter, _converterManager, createListener)`.

we can add passing descriptor to `BindingFactory.GetTriggerBinding`. But it will require SDK a change,